consensus/src/block_storage/block_store.rs

```diff
@@ -556,7 +556,7 @@ impl BlockStore {
         counters::CONSENSUS_PROPOSAL_PENDING_ROUNDS.observe(pending_rounds as f64);
         counters::CONSENSUS_PROPOSAL_PENDING_DURATION.observe(latency_to_committed.as_secs_f64());
 
-        latency_to_committed
+        latency_to_committed.saturating_sub(latency_to_ordered_ms)
```

Comment:
LGTM, can you add some comments in the function explaining what latency is it calculating and justification behind that? Also, should we change the name of the function to `pipeline_ordered_to_committed_latency` to make it clearer?